conversion/biostudies/bio_study_converter.py

```diff
@@ -79,5 +79,13 @@ def convert_all_studies(studies: List[Entity]):
 
     @staticmethod
     def convert_study(entity: Entity):
+        submission_payload = JsonMapper(entity.attributes).map(BIO_STUDY_SPEC)
+        return BioStudyConverter.__add_accession(entity, submission_payload)
 
-        return JsonMapper(entity.attributes).map(BIO_STUDY_SPEC)
+    @staticmethod
+    def __add_accession(study: Entity, submission_payload: dict):
+        study_accession: str = study.get_accession('BioStudies')
+        if study_accession:
+            submission_payload['accno'] = study_accession
+
+        return submission_payload
```

Comment:
Because of the way dictionaries work in python (and I think all variables)
You don't need to return the submission payload from the `__add_accession method`

```bio_study_converter.py 
@staticmethod
def convert_study(entity: Entity):
    submission_payload = JsonMapper(entity.attributes).map(BIO_STUDY_SPEC)
    BioStudyConverter.__add_accession(entity, submission_payload)
    return submission_payload

@staticmethod
def __add_accession(study: Entity, submission_payload: dict):
    study_accession: str = study.get_accession('BioStudies')
        if study_accession:
            submission_payload['accno'] = study_accession

```

Reply:
I know that, but it does not matter where that return statement is. Either in the `__add_accession` or the `convert_study` method. Your version clearly shows there is a side effect happening.